SPECS/glib/glib.spec

```diff
@@ -99,7 +102,6 @@ make DESTDIR=%{buildroot} install
 %{_bindir}/*
 %{_libdir}/*.a
 %{_libdir}/*.so
-%{_libdir}/*.la
```

Comment:
Why is this file gone now? I don't think anyone needs it, just curious.

Reply:
I believe this is due to the change of the implementation of different versions of glib. In the previous version, I can see the libtool libraries being defined in the Makefile, whereas in the newer version, they no longer exist and thus no *.la files will be created